src/mds/Locker.h

```diff
@@ -204,11 +204,12 @@ class Locker {
 
   // Maintain a global list to quickly find if any caps are late revoking
   xlist<Capability*> revoking_caps;
-  // Maintain a per-client list to find clients responsible for late ones quickly
-  std::map<client_t, xlist<Capability*> > revoking_caps_by_client;
```

Comment:
please add a public function that checks if the given client is in revoking_caps_by_client instead